qt/scientific_interfaces/Indirect/IndirectFitPlotModel.cpp

```diff
@@ -276,8 +277,21 @@ bool IndirectFitPlotModel::canCalculateGuess() const {
     return false;
 
   const auto composite = std::dynamic_pointer_cast<CompositeFunction>(function);
+  const auto resolutionLoaded = isResolutionLoaded();
   const auto isEmptyModel = composite && composite->nFunctions() == 0;
-  return getWorkspace() && !isEmptyModel;
+  return getWorkspace() && !isEmptyModel && resolutionLoaded;
+}
+
+bool IndirectFitPlotModel::isResolutionLoaded() const {
+  const auto model = dynamic_cast<ConvFitModel *>(m_fittingModel);
+  if (model) {
+    if (m_fittingModel->getResolutionsForFit().size() != 0) {
+      return true;
+    } else {
+      return false;
```

Comment:
This looks like it could just be:
```suggestion
    return m_fittingModel->getResolutionsForFit().size() != 0;
```

Reply:
Yep good point, thanks!